source/jsonnet/england-wales/individual/blocks/employment/travel_to_work.jsonnet

```
@@ -1,12 +1,30 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title, description) = {
-  title: title,
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'How does <em>{person_name}</em> usually travel to work?',
+    placeholders: [
+      placeholders.personName(),
+    ],
+  }
+  else 'How do you usually travel to work?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'Answer for the longest part, <em>by distance</em>, of their current journey to work',
+    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
+  ] else [
+    'Answer for the longest part, <em>by distance</em>, of your current journey to work',
```

I can see these were wrong prior to the changes, but these should end with a full stop.
```suggestion
    'Answer for the longest part, <em>by distance</em>, of their current journey to work.',
    'If the <strong>coronavirus</strong> pandemic has affected how they travel to work, select the answer that best describes their <strong>current circumstances</strong>.',
  ] else [
    'Answer for the longest part, <em>by distance</em>, of your current journey to work.',
```